moped-editor/src/views/projects/projectView/ProjectTeamTable.js

```diff
@@ -42,7 +42,7 @@ const ProjectTeamTable = ({
   const classes = useStyles();
 
   const { loading, error, data, refetch } = useQuery(TEAM_QUERY, {
-    variables: { projectId },
+    variables: { projectId: projectId ?? 0 },
```

Comment:
👍

Reply:
maybe it doesn't hurt to add a comment above this line that a `null` value bombs the app and help explains the fallback value of 0.